dev/core/src/com/google/gwt/dev/js/JsToStringGenerationVisitor.java

```diff
@@ -84,6 +84,7 @@
 /**
  * Produces text output from a JavaScript AST.
  */
+@SuppressWarnings("checkstyle.MethodName")
```

Comment:
Maybe we land these other checkstyle changes in a separate, earlier PR?

The one thing thats nice about the comments for a 1000line class like this is that only a third of the file needs special treatment, though I'm sure we don't want to add dozens of annotations. 

...do we really save that much effort by just using `_` instead of a prefix like `print` or `p`? This seems silly.

This is definitely load-bearing code though, this is used to generate our actual output JS, so some caution is warranted.